testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -190,24 +214,33 @@ object ServiceTest {
      */
     def ssl: Boolean
 
+    /**
+     * The server HTTPS port.
+     */
+    def sslPort: Int
```

Comment:
```suggestion
    def sslPort: Option[Int]
```

Maybe make this an option? Then `def ssl: Boolean` becomes `def ssl: Boolean = sslPort.isDefined`?